src/components/ModalWithImage.js

```diff
@@ -17,9 +17,5 @@ export default class ModalWithImage extends Modal {
 
   setEventListeners() {
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.